GlotPress/GlotPress-WP
# Add a filter for the classes of the translation table and an action after the translation table is displayed

## Problem

If we need to filter the classes of the translation table, or we need to add some element after the translation table is displayed, we can't do it. 

This PR is related with [this one](https://github.com/WordPress/wordpress.org/pull/169) at translate.wordpress.org.


## Solution

This PR adds: 
- a filter tochange the classes of the translation table. 
- an action to add content after the translation table is displayed.



## Testing Instructions

You can check this PR with [this other PR](https://github.com/WordPress/wordpress.org/pull/169), so you will see a live preview for the patterns on the right side of the translations table.
